analyzers/slither/.deepsource/issues/SLITHER-W1016.toml

```diff
@@ -0,0 +1,19 @@
+title = "Use Codex to find vulnerabilities"
```

Comment:
If I understand this correctly, this rule is using codex to find vulnerabilities, right?

This should be named `Vulnerabilities detected by codex` if that's correct.

Reply:
Should we be changing the issue titles? Isn't it better to keep them as they are in the tool to avoid confusing the user?

Reply:
Hm. Let this be.